README.md

```diff
@@ -57,3 +57,39 @@ If the file file you selected is supported by Nextcloud Text, Nextcloud Office o
 you will be able to collaborate on this file with all the call participants.
 
 You can then press "Close for all" if you want to hide the app for all participants.
+
+## Copy link to Clipboard and close the window/iframe
+
+This button will copy the desired link and can close the picker app window if it has been opened with a js `pickerWindow` button like 
+```
+newButton.onclick = () => {
+  pickerWindow = window.open('https://your.nextcloud.org/index.php/apps/picker/single-link?option=Clipboard', 'pickerWindow', 'popup');
```

Comment:
```suggestion
  pickerWindow = window.open('https://example.com/index.php/apps/picker/single-link?option=Clipboard', 'pickerWindow', 'popup');
```
Please always use designated example URLs instead of crafting your own custom one.